### DISCUSSION TITLE
2.6.13 更新日志

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#3086] [讨论]
· Mod 管理的搜索功能优化
　　极大地加快展示搜索结果的速度，现在应该几乎感受不到卡顿了
　　搜索结果会显示搜索到的 Mod 的数量和启用、禁用状态
　　去除开始、结束搜索的页面切换动画
· 将 Mojang 账户迁移入口改为官网入口（#3022，@Mcenahle）
· 下载 Mod 时支持筛选和显示 NeoForge（#3074，@iPodtouch6557）
· 显示 Mod/整合包 支持的 MC 版本时不再考虑 1.9 及更早的版本
· 修复了以下 11 个 Bug（包含 1 个严重 Bug）
　　[严重] 无法启动 Forge 1.20.3+
　　特定系统设置下启动器内时间格式显示异常（#178，@slientaurora）
　　取消更多子页面勾选会错误地勾选最后两项（#3019，@FQO3）
　　特定情况下主页设置被错误地重置为空白（#3021，@xiaobaimc）
　　在 Mod 管理页面搜索时会重新尝试获取图标（#3029，@Harvey2433）
　　部分页面没有返回顶部按钮（#3038）
　　Mod 信息可能显示出 CurseForge 删除的项目（#3041，@HowardZHY）
　　主页预设设置总是显示为“你知道吗？”（#3061，@JingHai-Lingyun）
　　第三方登录的账号密码有特殊字符会导致登录失败（#3063，@Azusa-mikan）
　　Mod 下载页面的过滤条件在特定操作下出现布局错误（#3067，@allMagicNB）
　　无法加载个别 CurseForge Mod 图标（#3075，@kong-que-dnf123）

<hr /><em>This discussion was created from the release <a href='https://github.com/Hex-Dragon/PCL2/releases/tag/'>2.6.13</a>.</em>

**Comment by allMagicNB** [comment]
占个位（
龙猫早点睡吧，对自己好点，别年纪轻轻大有前途的，给自己熬坏了（不是

**Comment by Azusa-mikan** [comment]
辛苦了

**Comment by Azusa-mikan** [comment]
现在才发现可以解锁一个主题（）
咱满足要求了（）
@LTCatt

**Reply by hejiehao** [reply]
在已完成的 Issue 中发送识别码
